### ISSUE TITLE
Add feedback after copying incident code

### ISSUE THREAD

**Issue by noah-schechter** [atlosdotorg/atlos#472]
It would be really nice to add some sort of small feedback indicating that the user has successfully copied the incident code to their clipboard; I wasn't sure if this was successful until I tried pasting:

https://user-images.githubusercontent.com/100018299/214412059-bf794f1d-8258-483f-9f7f-b2a89f5bc242.mov

**Comment by milesmcc** [comment]
Yeah some kind of little unobtrusive notification — Miguel got annoyed by our old popup feedback.
